[2021-09-07 04:45:25,509] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: users_and_posts_etl_dag.load_data 2021-09-07T04:45:23.908068+00:00 [queued]>
[2021-09-07 04:45:25,519] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: users_and_posts_etl_dag.load_data 2021-09-07T04:45:23.908068+00:00 [queued]>
[2021-09-07 04:45:25,519] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-09-07 04:45:25,519] {taskinstance.py:1095} INFO - Starting attempt 1 of 1
[2021-09-07 04:45:25,519] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-09-07 04:45:25,531] {taskinstance.py:1114} INFO - Executing <Task(PythonOperator): load_data> on 2021-09-07T04:45:23.908068+00:00
[2021-09-07 04:45:25,534] {standard_task_runner.py:52} INFO - Started process 22848 to run task
[2021-09-07 04:45:25,537] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'users_and_posts_etl_dag', 'load_data', '2021-09-07T04:45:23.908068+00:00', '--job-id', '186', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp33y43w0u', '--error-file', '/tmp/tmpvo5b60_m']
[2021-09-07 04:45:25,537] {standard_task_runner.py:77} INFO - Job 186: Subtask load_data
[2021-09-07 04:45:25,572] {logging_mixin.py:109} INFO - Running <TaskInstance: users_and_posts_etl_dag.load_data 2021-09-07T04:45:23.908068+00:00 [running]> on host 731700afc5ff
[2021-09-07 04:45:25,611] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=users_and_posts_etl_dag
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2021-09-07T04:45:23.908068+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-07T04:45:23.908068+00:00
[2021-09-07 04:45:25,629] {base.py:70} INFO - Using connection to: id: nickel-aws. Host: nickel.caybmxchvlv1.us-east-1.rds.amazonaws.com, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-09-07 04:45:26,479] {base.py:70} INFO - Using connection to: id: nickel-aws. Host: nickel.caybmxchvlv1.us-east-1.rds.amazonaws.com, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-09-07 04:45:27,427] {python.py:151} INFO - Done. Returned value was: None
[2021-09-07 04:45:27,443] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=users_and_posts_etl_dag, task_id=load_data, execution_date=20210907T044523, start_date=20210907T044525, end_date=20210907T044527
[2021-09-07 04:45:27,474] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-07 04:45:27,499] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
